tests/release/release.go

```diff
@@ -44,6 +49,9 @@ var _ = framework.ReleaseSuiteDescribe("test-demo", func() {
 	var devNamespace = uuid.New().String()
 	var managedNamespace = uuid.New().String()
 
+	var failure1SourceNamespace = "user-" + uuid.New().String()
```

Comment:
Why are the application name, source ReleaseLink name and release name constants but not these? I also don't think we should use `failureX` as part of the string name. I think this line would be better written `var failureMissingReleaseLinkSourceNamespace = "dev-" + uuid.New().String()` the name is kind of long but I think it is appropriate